aspnetcore/fundamentals/minimal-apis/min-api-filters/7samples/Filters/Program.cs

```diff
@@ -0,0 +1,108 @@
+#define ABC // FIRST SECOND ABC XYZ
+#if NEVER
+#elif FIRST
+#region snippet1
+var builder = WebApplication.CreateBuilder(args);
+
+var app = builder.Build();
+
+string ColorName(string color) => $"Color specified: {color}!";
+
+app.MapGet("/colorSelector/{color}", ColorName)
+    .AddFilter(async (rhiContext, next) =>
```

Comment:
```suggestion
    .AddFilter(async (invocationContext, next) =>
```

Nit: I think including the full world "invocation" makes it a little easier to grok that this is the per-invocation context as opposed to the handler context provided in the filter factory delegate.